CWE/inventions/New inventions.txt

```diff
@@ -300,4 +300,63 @@ full_body_replacement = {
             health = 0.05
         }
 	}
+}
+
+land_ethic = {
+	limit = { intensive_farming = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 2
+			year = 1980
+		}
+	}
+
+	effect = {
+		shared_prestige = 5
+		farm_rgo_eff = 0.01 #More efficient/sustainable use of land
+	}
+}
+
+smart_grid = {
+	limit = { sustainable_energy = 1 }
+	chance = {
+		base = 1
+		modifier = {
+			factor = 2
+			handheld_devices = 1
+		}
+		modifier = {
+			factor = 2
+			modern_backbone = 1
+		}
+	}
+	effect = {
+		factory_goods_throughput = {
+			heating_electricity = 0.05 #I think this reduces the amount of input to make an output?
+		}
+	}
+}
+
+techno_progressivism #Techno-progressivism
+#Techno-progressivism is an ideology mixture of transhumanism and progressive thought that seeks to pursue technocracy within the framework of liberal democracy - with a focus on accountability and transparency.
```

Comment:
This one is broken by comments

Reply:
I am aware - its a wip. Only making it a draft so other people can contribute. There were a few people on the server that said they wanted to add inventions and participate in tweaking them so I am hoping by making this a draft people can easily add to it via pull requests to my branch instead of having to send the files and have me add them.